src/Shared/Data/ServiceInsightsDbContext.cs

```diff
@@ -12,74 +12,249 @@ public ServiceInsightsDbContext(DbContextOptions<ServiceInsightsDbContext> optio
     {
     }
 
+    public virtual DbSet<Analytic> Analytics { get; set; }
```

Comment:
Don't think we need this. I think you might have unintentionally scaffolded this with Entity Framework. Try deleting the table and re-scaffolding